samba-integration-centos-ci-tests.sh

```diff
@@ -17,26 +17,31 @@ SCRIPT_GIT_BRANCH="centos-ci"
 SCRIPT_NAME="$(basename $0)"
 SCRIPT_PATH="$(realpath $0)"
 
-# enable additional sources for yum
-# (SCL repository for Vagrant, epel for ansible)
-yum -y install centos-release-scl epel-release
+# enable additional sources for yum:
+# - epel for ansible
+yum -y install epel-release
 
 # Install additional packages
 #
-# note: adding sclo-vagrant1-vagrant explicitly seems to fix
-#   issues where libvirt fails to bring up the vm with errors like this:
-#   "Call to virDomainCreateWithFlags failed: the CPU is incompatible with host
-#    CPU: Host CPU does not provide required features: svm" (or vmx)
-#
 yum -y install \
 	qemu-kvm \
 	qemu-kvm-tools \
 	qemu-img \
-	sclo-vagrant1-vagrant \
-	sclo-vagrant1-vagrant-libvirt \
 	git \
 	make \
-	ansible
+	ansible \
+	libvirt \
+	libvirt-devel
+
+yum -y group install "Development Tools"
+
+if ! rpm -q vagrant
```

Comment:
What are we anticipating here? Is it like _vagrant_ might get installed by default with CentOS box? Or as part of some dependency for other packages getting installed?

Reply:
yeah, I was wondering the same. Took it from the heketi patch. will remove here :-)

Reply:
Hmmm, it now failed with:

```
19:27:36 + yum -y install https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
19:27:36 Loaded plugins: fastestmirror
19:27:37 Examining /var/tmp/yum-root-5IF4Zr/vagrant_2.2.7_x86_64.rpm: 1:vagrant-2.2.7-1.x86_64
19:27:37 /var/tmp/yum-root-5IF4Zr/vagrant_2.2.7_x86_64.rpm: does not update installed package.
19:27:37 Error: Nothing to do
19:27:37 make: *** [test] Error 1
```

I think it is related to us restarting the script w/o the PR . Checking it and will update...